src/main/java/oxgame/app/RoundBuffer.java

```diff
@@ -17,10 +17,9 @@ public void addPlayer(Player player) throws UnsupportedOperationException{
 
     public Player takePlayer() {
         Player resultPlayer = players[0];
```

Comment:
Minor issue here, just return players[0] without assigning

Reply:
Ready, will be in next commit.